lib/gemstash/configuration.rb

```diff
@@ -20,7 +20,13 @@ class Configuration
       puma_threads: 16,
       puma_workers: 1,
       cache_expiration: 30 * 60,
-      cache_max_size: 500
+      cache_max_size: 500,
+      storage_adapter: "local",
```

Comment:
All this new config will need documentation here https://github.com/rubygems/gemstash/blob/master/docs/gemstash-customize.7.md

Reply:
Added in c072f9d.